components/Dashboard/Dashboard.js

```diff
@@ -18,15 +25,99 @@ export default function Dashboard() {
     setSelectedColumn(column);
   }
 
+  const { allEntries, isLoadingEntries, errorEntries } =
+    useData().fetchedAllEntries;
+  const { userEntries } = useData().fetchedUserEntries;
```

Comment:
its just a small thing but you are combining two ways of accessing object values here, a bit cleaner would be to just use one: 

```
const userEntries = useData().fetchedUserEntries.userEntries

const { fetchedUserEntries: { userEntries } }= useData()
```

Reply:
Danke! Dein Vorschlag wirft einen Fehler aus weil userEntries 2x deklariert wird.

Reply:
I added two examples you can pick one of them.